### PR TITLE
Fixing pipeline and make pb imports to be done in only one file

### PR DESCRIPTION
**Pull request by robinbernon**
Made a temp fix for our workflow. (Have spoken to SF and they said they will make a patch for the next release so we can set the CI back to what is was previously for the next release.

Also got pb binding imports to be handled in just one file rather than a folder which was used previously.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -43,4 +44,7 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
       - name: build all
-        run: make build-all
+        run: |
+          chmod a+x /opt/hostedtoolcache/streamingfast/substreams/v0.0.21/linux-x64/substreams_0.0.21_linux_x86_64/substreams
```
Comment (this-username-is-taken) [inline .github/workflows/ci.yaml:48]: any way we can avoid hardcoding this?

Comment (robinbernon) [reply .github/workflows/ci.yaml:48]: Don't think so. I have fixed the version for the substreams CLI to v21 so it definitely won't start breaking when they make the next release. And once we see they've made the first release with the fix to allow for proper unpacking of the binary during the installation stage we can revert the code back.

Comment (this-username-is-taken) [reply .github/workflows/ci.yaml:48]: gotcha, sounds good

**Review by this-username-is-taken — APPROVED** [review]
Looks great! Thanks for digging into the issue!

**Pull request merged**